StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift

```diff
@@ -0,0 +1,308 @@
+                                                      right:  152))
+    }
+    
+    @objc public func toggleButtons() {
+        guard let approveButton = approveButton, let reproveButton = reproveButton  else { return}
+        
+        if (!notApplicableSwitch!.isOn) {
+            approveButton.removeTarget(self, action: #selector(warningAction), for: .touchUpInside)
+            approveButton.addTarget(delegate, action: #selector(delegate?.goToAddApproveInspectionData), for: .touchUpInside)
+            approveButton.backgroundColor = .greenStant
+            reproveButton.removeTarget(self, action: #selector(warningAction), for: .touchUpInside)
+            reproveButton.addTarget(delegate, action: #selector(delegate?.goToAddReproveInspectionData), for: .touchUpInside)
+            reproveButton.backgroundColor = .redLightStant
+        } else {
+            approveButton.removeTarget(delegate, action: #selector(delegate?.goToAddApproveInspectionData), for: .touchUpInside)
+            approveButton.addTarget(self, action: #selector(warningAction), for: .touchUpInside)
+            approveButton.backgroundColor = .darkGrayStant
+            reproveButton.removeTarget(delegate, action: #selector(delegate?.goToAddReproveInspectionData), for: .touchUpInside)
+            reproveButton.addTarget(self, action: #selector(warningAction), for: .touchUpInside)
+            reproveButton.backgroundColor = .darkGrayStant
+        }
+    }
+    
+    @objc func warningAction() {
+        let alert = UIAlertController(title:          AppStrings.verified_method_details_warning_title_action,
+                                      message:        AppStrings.verified_method_details_warning_message_action,
+                                      preferredStyle: .alert)
+            
+            alert.addAction(UIAlertAction(title:   AppStrings.verified_method_details_warning_button_ok ,
+                                          style:   .default,
+                                          handler: nil))
+            
+            if let present = delegate {
+                   present.present(alert: alert)
+            }
+    }
+
+    fileprivate func addShadow() {
+        self.layer.applySketchShadow(color:  .shadowStant,
+                                     alpha:  0.09,
+                                     x:      0,
+                                     y:      3,
+                                     blur:   8,
+                                     spread: 0)
+    }
+    
+    public func makeShadow(button: UIButton) {
+        button.layer.shadowPath    = UIBezierPath(rect: button.bounds).cgPath
+        button.layer.shadowRadius  = 4
+        button.layer.shadowOffset  = CGSize(width: 0, height: 3)
+        button.layer.shadowOpacity = 0.1
+    }
+}
+
+@objc public protocol VerifiedMethodDetailViewDelegate {
+    @objc func goToAddApproveInspectionData()
+    @objc func goToAddReproveInspectionData()
```

Comment:
Change the name of this function. It should be goToReproveVerifiedMethod

Reply:
Done